tests/ios/app/snapshot_test.m

```diff
@@ -0,0 +1,17 @@
+@import XCTest;
+#import <BazelRunfileBundleBridge/BazelRunfileBundleBridge.h>
+
+@interface EmptyTests : XCTestCase
+
+@end
+
+@implementation EmptyTests
+
+- (void)testFailureEndToEnd
+{
+    // TODO: update for snapshots trees
+    // For now, we load the actual test bundle
+    XCTAssertNonNil([[NSBundle mainBundle] bazelRunfilePathForResource:@"build_bazel_rules_ios/tests/ios/app/TestSnapshots"]);
```

Comment:
iOS developers may not find it idiomatic to pass in the path of their BUILD file in like this. Perhaps we can hack this in somehow so they can call:
```
    XCTAssertNonNil([[NSBundle mainBundle] bazelRunfilePathForResource:@"ReferenceImages"]);
```

Otherwise we can add an API like this:
```
-[NSBundle testReferenceImages]
```